src/shared/components/GenericList/SearchInput.js

```diff
@@ -103,19 +103,23 @@ export function SearchInput({
   };
 
   return (
-    <Input
-      id="search-input"
-      accessibleName={`search-${entriesKind}`}
-      role="search"
-      type="Text"
-      ref={searchInputRef}
-      placeholder={t('common.tooltips.search')}
-      className="search-with-magnifying-glass"
-      value={searchQuery}
-      onInput={e => handleQueryChange(e.target.value)}
-      showSuggestions={showSuggestion}
-    >
-      {showSuggestion && renderSearchList(filteredEntries)}
-    </Input>
+    <div className="search-input-container">
```

Comment:
I think that it would be easier to maintain and understand just to add: 
```
            <Label for="search-input" showColon>
        {t('common.tooltips.search')}
      </Label>
```
to:
- https://github.com/kyma-project/busola/blob/main/src/shared/components/GenericList/GenericList.js#L162
- https://github.com/kyma-project/busola/blob/main/src/resources/Pods/ContainersLogs.js#L206
- https://github.com/kyma-project/busola/blob/main/src/components/CustomResources/GroupingListPage.js#L104
According to the: `duplication is far cheaper than the wrong abstraction`

Reply:
For Grouping list I created someting like that:
```
        actions={
          <FlexBox alignItems={"Center"} style={{gap: "0.5rem"}}>
          <Label for="search-input" showColon>
            {t('common.tooltips.search')}
          </Label>
          <SearchInput
              value={searchQuery}
              handleQueryChange={setSearchQuery}
              allowSlashShortcut
          />
            </FlexBox>
        }
```

Reply:
The thing is, to ensure responsiveness, both the label and the input need to be wrapped in a container:
<img width="344" alt="image" src="https://github.com/user-attachments/assets/ac28ad04-9f74-454c-bbc8-90dca073c23c">
I think handling this in a single component makes more sense tbh